identity_core/src/did.rs

```diff
@@ -252,3 +211,11 @@ impl Serialize for DID {
         serializer.serialize_str(s.as_str())
     }
 }
+
+impl From<DIDTuple> for Param {
+    fn from(tuple: DIDTuple) -> Param {
+        let (name, value) = tuple;
```

Comment:
💅 you could use `(name, value)` in the function signaure: `fn from((name, value): DIDTuple) -> Param`

Reply:
Ahh yes, you are right, should have just destructed from the arguments.